.evergreen.yml

```diff
@@ -26,7 +26,13 @@ functions:
         . venv/bin/activate
         pip install --upgrade pip
 
-        pip install poetry
+        pip install pipx
+
+        export PIPX_HOME="$HOME/pipx"
+        export PIPX_BIN_DIR="$HOME/pipx/bin"
+        export PATH="$PATH:$PIPX_BIN_DIR"
+
+        pipx install poetry
```

Comment:
Can you pin pipx and poetry like we do [here](https://github.com/10gen/build-baron-tools/blob/master/.evergreen.yml#L85)? That way we can better insure against breakages from new versions of them.

Reply:
done